server/@types/bapv.d.ts

```diff
@@ -139,6 +139,7 @@ export type VisitSlot = {
   startTimestamp: string
   endTimestamp: string
   availableTables: number
+  capacity: number
```

Comment:
Adding capacity to `visitSlot` as it wasn't actually available on the date/time page - this results in most of the changes in this PR because of updates to test data, etc.